### DISCUSSION TITLE
How to add free text input?

### DISCUSSION THREAD

**Discussion by basicx-shivani** [atoti/atoti#535] [Q&A] [answered]
I want to create input filter as free text to user write and then proceed. how to create it in atoti?

**Comment by tibdex** [comment, marked as answer]
In the upcoming release, the UI will be customizable with app extensions. This will be an [Atoti+](https://docs.atoti.io/0.6.4/atoti_plus.html) feature. The [`atoti-plus` plugin](https://docs.atoti.io/0.6.4/lib/atoti-plus/atoti_plus.html#module-atoti_plus) will actually come with an extension that adds a rich-text editor widget.

**Reply by tibdex** [reply]
See https://github.com/atoti/atoti/discussions/544.

**Comment by basicx-shivani** [comment]
Hey, Atoti Team,
Thank you so much for quick response.
when can I expect new release so I will bind up my dashboard according 
to this.
Thanks,
Shivani

On 2022-03-23 10:47, Thibault Derousseaux wrote:
> In the upcoming release, the UI will be customizable with app
> extensions. This will be an Atoti+ [1] feature. The atoti-plus plugin
> [2] will actually come with an extension that adds a rich-text editor
> widget.
> 
> --
> Reply to this email directly, view it on GitHub [3], or unsubscribe
> [4].
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> 
> Links:
> ------
> [1] https://docs.atoti.io/0.6.4/atoti_plus.html
> [2] 
> https://docs.atoti.io/0.6.4/lib/atoti-plus/atoti_plus.html#module-atoti_plus
> [3] 
> https://github.com/atoti/atoti/discussions/535#discussioncomment-2421635
> [4] 
> https://github.com/notifications/unsubscribe-auth/ATVBOXTP2GHLN6ORFG4XAQ3VBLZFHANCNFSM5RNSWU7A

-- 
Shivani Jagani
Python  Developer
E-Mail: ***@***.***

StartWith Basicx Pvt. Ltd
Jaipur, Rajasthan (In)
www.basicx.in

**Reply by tibdex** [reply]
[The release is out](https://docs.atoti.io/0.6.5/releases/0.6.5.html).
